crates/orchestrator/src/generators/chain_spec.rs

```diff
@@ -392,147 +392,224 @@ impl ChainSpec {
         };
 
         if (self.runtime.is_some() && self.asset_location.is_none()) || is_omni_node {
```

Comment:
We should remove this workaround. The idea is to follow this flow:
- First try to create the raw spec using `GenericChainSpec`
- IFF fails with the expected errors 
  - Fallback to use the cmd logic